src/main/java/com/hm/achievement/AdvancedAchievements.java

```diff
@@ -349,19 +350,28 @@ public void extractConfigurationParameters() {
 	 * @throws PluginLoadError
 	 */
 	public CommentedYamlConfiguration loadAndBackupFile(String fileName) throws PluginLoadError {
```

Comment:
The method was quite small, was it really useful to break it up? ^^

Reply:
At the time of split I was still implementing reloading for the new LanguageConfig and didn't know if I would need to call backupFile separately - Now that the class is gone I reverted the split.